_usage/locales.md

```diff
@@ -169,72 +177,71 @@ Babelfish collations.
+in SQL Server.
 
 
 #### Tertiary collations
 
-SQL collations, such as `SQL_Latin1_General_Pref_CP1_CI_AS`, support
+SQL collations, such as `sql_latin1_general_pref_cp1_ci_as`, support
 the `TERTIARY_WEIGHTS` function and the ability to sort strings that
-compare equally in a `CI_AS` collation to be sorted upper case
+compare equally in a `ci_as` collation to be sorted upper case
 first: `ABC`, `ABc`, `AbC`, `Abc`, `aBC`, `aBc`, `abC`, and finally
 `abc`. Thus the `DENSE_RANK() OVER (ORDER BY column)` analytic
 function would assess these strings as having the same rank but
 would order them upper case first within a partition.
 
 A similar result can be obtained with Babelfish by adding a
 `COLLATE` clause to the `ORDER BY` clause that specifies a tertiary
-`CS_AS` collation that specifies `@colCaseFirst=upper`.
+`cs_as` collation that specifies `@colCaseFirst=upper`.
 Unfortunately the `colCaseFirst` modifier applies only to strings
-that are tertiary-equal (rather than secondary-equal like a `CI_AS`
+that are tertiary-equal (rather than secondary-equal like a `ci_as`
 collation); therefore, tertiary SQL collations cannot be emulated
-using a single ICU collation. As a workaround, we recommend you
-modify applications that use the `SQL_Latin1_General_Pref_CP1_CI_AS`
-collation to use the `BBF_SQL_Latin1_General_CP1_CI_AS` collation
-first, and then add `COLLATE BBF_SQL_Latin1_General_Pref_CP1_CS_AS`
-to any `ORDER BY` clause for this column.
+using a single ICU collation. As a workaround, we recommend that
+you use a different collation for `GROUP BY` than you use for `ORDER BY`.
 
-#### PostgreSQL uses a specific version of ICU and can match at most one
-
-version of a collation. Variations across versions are unavoidable,
-as are minor variations across time as languages evolve.
+#### PostgreSQL supports exactly one version of each collation
 
+PostgreSQL supports exactly one version of each collation, whereas SQL
+Server can support multiple versions of the same collation, typically
+by modifying the collation name with a version suffix such as `_80` or `_100`.
 
 #### Character(SC) expansion
 
 A character expansion treats a single character as equal to a
-sequence of characters at the primary level. SQL Server\'s default
-`CI_AS` collation supports character expansion, whereas ICU
+sequence of characters at the primary level. SQL Server's default
+`ci_as` collation supports character expansion, whereas ICU
 collations support character expansion only for accent-insensitive
 collations.
 
 Character reordering options are currently more restricted in
-PostgreSQL than in ICU, and the order of special characters can\'t
+PostgreSQL than in ICU, and the order of special characters can't
 be made to follow the order of special characters in SQL Server
 exactly.
 
-When character expansion is required, then a `AI` collation must be
+When character expansion is required, then an `AI` collation must be
```

Comment:
for consistency, use the same case everywhere, i.e. 'ai'

Reply:
I changed all occurrences to lower case.